targets/TARGET_Samsung/TARGET_SIDK_S5JS100/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if(${MBED_TOOLCHAIN} STREQUAL "ARM")
+    set(LINKER_FILE device/TOOLCHAIN_ARM_STD/sidk_s5js100.sct)
+    set(STARTUP_FILE device/TOOLCHAIN_ARM_STD/startup_sidk_s5js100.S)
+elseif(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    set(LINKER_FILE device/TOOLCHAIN_GCC_ARM/sidk_s5js100.ld)
+    set(STARTUP_FILE device/TOOLCHAIN_GCC_ARM/startup_sidk_s5js100.S)
+endif()
+
+set_property(GLOBAL PROPERTY MBED_TARGET_LINKER_FILE ${CMAKE_CURRENT_SOURCE_DIR}/${LINKER_FILE})
+
+target_include_directories(mbed-core
+    INTERFACE
+        .
+        device
+        modem
+        security_subsystem
+        security_subsystem/drivers
+)
+
+target_sources(mbed-core
+    INTERFACE
+# TODO CMAKE: The files commented below trigger a dependency on other libraries
```

Comment:
I've missed this one. I created #14025 to track this

Reply:
@pan- is aware of this and has raised the issue with @ARMmbed/team-samsung.